### DISCUSSION TITLE
GitHub Actions added to all projects

### DISCUSSION THREAD

**Discussion by p-mongo** [mongodb/mongoid#5069] [Announcements]
Thanks to @johnnyshields  and @comandeo,  all Ruby driver team projects (Mongoid, [driver](https://github.com/mongodb/mongo-ruby-driver) and [bson-ruby](https://github.com/mongodb/bson-ruby)) now have GitHub actions configured.

All projects continue to use Evergreen as the final CI system, but GHA should cover the majority of configurations.

Please also note that all projects have test readmes (e.g. https://github.com/mongodb/mongoid/blob/master/spec/README.md) that provide basic instructions for executing the tests including the prerequisites.

Please let us know in this discussion if you have any feedback or would like GHA to cover additional configurations.

**Comment by johnnyshields** [comment]
Great work, I really appreciate it. This will allow us to iterate much faster, without need to paste spec failures into the threads!
